src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,66 @@
+l = [1, 5, 8, 4, 2, 9, 6, 0, 10, 3, 7]
+
+
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
     # TO-DO
-    
+    print(f"elements: {arrA} and {arrB} - merged_arr {merged_arr}")
+    i = 0
```

Comment:
Using these variable names gets a little confusing for me to read over. Generally i is an index and y z are graph coordinates, it may be better to use a and b which are common ways to compare two things, or more descriptive names.